catalog/spatial/csw/spatial-csw-transformer/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/transformer/GmdTransformer.java

```diff
@@ -95,14 +82,13 @@
 import ddf.catalog.data.types.Topic;
 import ddf.catalog.transform.CatalogTransformerException;
 import ddf.catalog.transform.InputTransformer;
-import ddf.catalog.transform.MetacardTransformer;
 
-public class GmdTransformer implements InputTransformer, MetacardTransformer {
-
-    private static final Logger LOGGER = LoggerFactory.getLogger(GmdTransformer.class);
+public class GmdTransformer extends AbstractGmdTransformer implements InputTransformer {
 
     public static final String GCO = "gco";
```

Comment:
do you think it makes sense to move these constants up to the abstract class?  I'm guessing some of these would be needed by any subclass of AbstractGmdTransformer